api/insight/apiroutes.go

```diff
@@ -41,31 +40,22 @@ type insightApiContext struct {
 	BlockData      *dcrpg.ChainDBRPC
 	params         *chaincfg.Params
 	MemPool        DataSourceLite
-	Status         apitypes.Status
+	status         *apitypes.Status
 	JSONIndent     string
 	ReqPerSecLimit float64
 	maxCSVAddrs    int
 }
 
 // NewInsightContext Constructor for insightApiContext
 func NewInsightContext(client *rpcclient.Client, blockData *dcrpg.ChainDBRPC, params *chaincfg.Params,
-	memPoolData DataSourceLite, JSONIndent string, maxAddrs int) *insightApiContext {
-	conns, _ := client.GetConnectionCount()
-	nodeHeight, _ := client.GetBlockCount()
-	version := semver.NewSemver(1, 0, 0)
```

Comment:
This version appeared to be unused.

Reply:
I don't think it's useful here.  Good to remove.